fe/fe-core/src/main/java/org/apache/doris/analysis/ShowBuildIndexStmt.java

```diff
@@ -136,7 +136,7 @@ public void analyze(Analyzer analyzer) throws AnalysisException, UserException {
 
     private void analyzeSubPredicate(Expr subExpr) throws AnalysisException {
         if (subExpr == null) {
```

Comment:
I think we can simply remove the `if` in line 99.
And not change here.

Reply:
BTW, could you please also modify the comment in line 44, that is a typo.
Thank you

Reply:
fixed